requirements.txt

```diff
@@ -18,7 +18,7 @@ dcplib >= 2.0.2, < 3
 
 # The version range of docutils is pinned by botocore. Pinning it here can cause a version conflict.
 # See https://github.com/boto/botocore/pull/1802, https://github.com/HumanCellAtlas/dcp-cli/issues/418
-docutils
+# docutils
```

Comment:
@kislyuk I'm unsure if this should be handled by commenting out the pinned requirement or removing ln21 entirely; i would like to keep the comment above, but don't want the un-pinning/removal to go unnoticed by another developer.  Any recommendation?

Reply:
This should probably be `docutils < 0.16` as per https://github.com/boto/botocore/commit/a8bbf60b665efee53f5a5d2cbbeb6e5e9e0838f2